lib/sass/file-to-class-names.ts

```diff
@@ -5,88 +5,76 @@ import { sourceToClassNames } from "./source-to-class-names";
 import { Implementations, getImplementation } from "../implementations";
 import { customImporters, Aliases, SASSImporterOptions } from "./importer";
 
+export { Aliases };
 export type ClassName = string;
-export type ClassNames = ClassName[];
+interface Transformer {
+  (className: ClassName): string;
+}
+
+const transformersMap = {
+  camel: (className: ClassName) => camelCase(className),
+  dashes: (className: ClassName) =>
+    /-/.test(className) ? camelCase(className) : className,
+  kebab: (className: ClassName) => transformersMap.param(className),
+  none: (className: ClassName) => className,
+  param: (className: ClassName) => paramCase(className),
+  snake: (className: ClassName) => snakeCase(className),
+} as const;
 
-export type NameFormat = typeof NAME_FORMATS[number];
+export type NameFormatInput = keyof typeof transformersMap | "all" | "none";
```

Comment:
`none` should be unnecessary here since it already exists in `transformersMap`

```suggestion
export type NameFormatInput = keyof typeof transformersMap | "all";
```

Reply:
Indeed you're right ... 👍  I'll remove it

Reply:
Addressed: [e67879f](https://github.com/skovy/typed-scss-modules/pull/164/commits/e67879fbfbadddc3b495be59a6183d29aa90b7bd)